app-backend/tile/src/it/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+gatling {
+  host = "https://app.staging.rasterfoundry.com"
+
+  refreshToken = ""
+  refreshToken = ${?REFRESH_TOKEN}
+
+  projectId = ""
+  projectId = ${?PROJECT_ID}
+
+  randomSeed = 42
+
+  tms {
+    template = "/tiles/${projectId}/$z/$x/$y?token=${authToken}"
```

Comment:
Yeah, it would be `?mapToken` -- however, if this still needs to query the API to get the bounding box for a project, then the refresh token may still be necessary.

We might need to think of a way to not hit the refresh token endpoint so hard though -- it should only be hit very sparingly because the token returned is good for a long time.